go/vt/sqlparser/parse_test.go

```diff
@@ -1054,6 +1055,15 @@ var (
 	}, {
 		input:  "load data from s3 'x.txt'",
 		output: "AST node missing for Load type",
+	}, {
+		input:  "load data from s3 manifest 'x.txt'",
+		output: "AST node missing for Load type",
+	}, {
+		input:  "load data from s3 file 'x.txt'",
+		output: "AST node missing for Load type",
+	}, {
+		input:  "load data infile 'x.txt' into table 'c'",
+		output: "AST node missing for Load type",
```

Comment:
What does this output format mean? We could do something similar to DDL formatting.

Reply:
we should never be calling format on load data queries but I still wanted to add tests for them in parsing

Reply:
I could add it as a separate test that does not check the output but only the error message.

Reply:
Moved the load data statements out into a separate test where I only check for errors and not the output.